specification/2.0/README.md

```diff
@@ -251,7 +251,7 @@ For property names, glTF uses [camel case](http://en.wikipedia.org/wiki/CamelCas
 
 ## Coordinate System and Units
 
-glTF uses a right-handed coordinate system, that is, the cross product of +X and +Y yields +Z. glTF defines +Y as up. The front of a glTF asset faces +Z.
+glTF uses a right-handed coordinate system; that is, the cross product of right and forward yields up. glTF defines +Y as up, +Z as forward, and -X as right.
```

Comment:
The line, "The front of a glTF asset faces +Z." is a useful clarification of "+Z as forward", as opposed to forward in some scene navigation or default camera orientation sense. I think I would keep that part. Otherwise looks good, thank you!

Reply:
Done.

Reply:
Had to read this through a couple times:

> ... the cross product of right and forward yields up ...

I think you mean `left ⨉ forward`? It may be better to stick with the original phrasing of this clause, "cross product of +X and +Y yields +Z". The right-hand rule does not really dictate the right/forward/up convention to my understanding (see various combinations here: https://www.evl.uic.edu/ralph/508S98/coordinates.html).

And thanks for correcting the comma. :)

Reply:
On second reading, my link above contradicts us:

> The default coordinate system in OpenGL(TM) is right-handed: the positive x and y axes point right and up, and the negative z axis points forward. 

This might need a closer review, @lexaknyazev would be interested in your thoughts.

Reply:
+X right and -Z forward is the right-handed system same as +X left and +Z forward, just rotated by 180 degrees.

Reply:
Ah thanks! I'm misreading things here. 

Sounds like this can be resolved.